### ISSUE TITLE
Geo-Engineering

### ISSUE THREAD

**Issue by Spunkmeister** [StrangeLoopGames/EcoSuggestions#1098]
I would really like to see Geo-Engineering as a legitimate option in the game. What I mean by that is some of the following:

- Re-routing rivers/lakes to water food fields
- 
- Creating dams to control water flows
- 
- Creating drydock area's on the ocean for building
- 
- Creating artificial lakes/ponds for water sourcing

Currently when you dig near a water source and attempt to channel water it doesn't follow the path to completion. Instead it just gets lower in height and stops after just a few blocks (of course this can be overcome by continually digging deeper) but, it prevents water routing from being viable.

**Comment by trickenso** [comment]
More water physics and options for terraforming will always get my vote.

FYI - You can prevent water from being absorbed into the ground using the 'aqueduct' brick block. This is viable for diverting rivers and lakes, and also for creating artificial lakes and ponds. It might not be ideal for all aesthetic purposes, but it's definitely possible already.

**Comment by Spunkmeister** [comment]
Yeah, I like that those are an option but, I really want the ability to terraform the land itself and create certain results with the natural landscape. I basically want something similar to the option included in the picture. 
![river divert](https://user-images.githubusercontent.com/64482810/80505838-de618000-8942-11ea-8d55-5ab6a0fc0a1b.jpeg)

**Comment by ThePiachu** [comment]
While geo-engineering sounds neat, it might be an issue with how small the worlds and features usually are. A dam for a river that's 5 blocks wide doesn't look that impressive. Might be neat to have some way of changing biomes around, maybe reclaiming some land from the sea and making it hospitable and what not...

**Comment by trickenso** [comment]
Biomes that change in response to the water source being moved would be glorious.
